airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -1956,23 +1956,85 @@ definitions:
+      type: object
+      examples:
+        - start_date: 2010-01-01
+          api_key: "*****"
     - title: headers
-      description: The HTTP headers from the last response received from the API.
+      description: The HTTP headers from the last response received from the API. The object's keys are the header names from the response.
+      type: object
+      examples:
+        - Server: nginx
+          Date: Mon, 24 Apr 2023 20:17:21 GMT
+          Content-Type: application/json
+          Content-Length: "420"
+          Connection: keep-alive
+          referrer-policy: strict-origin-when-cross-origin
+          x-content-type-options: nosniff
+          x-ratelimit-limit: "600"
+          x-ratelimit-remaining: "598"
+          x-ratelimit-reset: "39"
     - title: last_records
       description: List of records extracted from the last response received from the API.
+      type: list
+      examples:
+        - - name: "Test List: 19"
+            id: 0236d6d2
+            contact_count: 20
+            _metadata:
+              self: https://api.sendgrid.com/v3/marketing/lists/0236d6d2
+          - name: List for CI tests, number 30
+            id: 041ee031
+            contact_count: 0
+            _metadata:
+              self: https://api.sendgrid.com/v3/marketing/lists/041ee031
+    - title: next_page_token
+    - description: Object describing the token to fetch the next page of records. The object has a single key "next_page_token".
+      type: object
+      examples:
+        - next_page_token: 3
+        - next_page_token: https://api.sendgrid.com/v3/marketing/lists/0236d6d2-75d2-42c5-962d-603e0deaf8d1
     - title: record
-      description: The record being processed.
+      description: The record being processed. The object's keys are the same keys as the records produced by the RecordSelector.
+      type: object
     - title: response
-      description: The body of the last response received from the API.
+      description: The body of the last response received from the API. The object's keys are the same keys as the response body's.
+      type: object
+      examples:
+        - result:
+            - name: "Test List: 19"
+              id: 0236d6d2-75d2-42c5-962d-603e0deaf8d1
+              contact_count: 20
+              _metadata:
+                self: https://api.sendgrid.com/v3/marketing/lists/0236d6d2
+          _metadata:
+            self: https://api.sendgrid.com/v3/marketing/lists?page_size=1&page_token=
+            next: https://api.sendgrid.com/v3/marketing/lists?page_size=1&page_token=0236d6d2
+            count: 82
     - title: stream_interval
-      description:
+      description: The current stream interval being processed. The keys are defined by the incremental sync component. Default keys are `start_time` and `end_time`.
+      type: object
+      examples:
```

Comment:
While this example can theoretically happen, it will almost always be start_time and end_time as described above - could we reflect this in the example?

Reply:
thanks for fixing